app/charts/map/map.tsx

```diff
@@ -383,10 +383,7 @@ export const MapComponent = () => {
           {...viewState}
         >
           <div data-map-loaded={loaded} />
-          <DeckGLOverlay
-            interleaved
-            layers={[geoJsonLayer, scatterplotLayer]}
-          />
```

Comment:
Seems like you introduced it in response https://github.com/visualize-admin/visualization-tool/issues/713, could you quickly make sure that there is no problem ?

Reply:
https://github.com/visualize-admin/visualization-tool/pull/1019/commits/2f63295ecaa85fe5bbd49ba38aa5e4b355c03b49 🤞